hazelcast/src/main/java/com/hazelcast/ClientMain.java

```diff
@@ -11,8 +11,8 @@
 public class ClientMain {
 
     public static void main(String[] args) {
-        System.setProperty("hazelcast.tpc.enabled", "true");
-        System.setProperty("hazelcast.tpc.eventloop.count", "" + Runtime.getRuntime().availableProcessors());
+        System.setProperty("hz.alto.enabled", "true");
+        System.setProperty("hz.alto.eventloop-count", "" + Runtime.getRuntime().availableProcessors());
```

Comment:
@pveentjer did you see this? The defaults are configurable via system properties like this. Is this not enough?